Gordon360/Models/ViewModels/RecIM/ActivityExtendedViewModel.cs

```diff
@@ -27,28 +28,30 @@ public class ActivityExtendedViewModel
         public IEnumerable<TeamExtendedViewModel> Team { get; set; }
 
 
-        public static implicit operator ActivityExtendedViewModel(Activity a)
+        public static implicit operator ActivityExtendedViewModel(Activity? a)
```

Comment:
```suggestion
        public static implicit operator ActivityExtendedViewModel?(Activity? a)
```

You could also make the nullable version a separate function that propagates the null or else uses the non-nullable.